tests/api_tests.rs

```diff
@@ -1,99 +1,103 @@
+#![feature(async_closure)]
```

Comment:
What's the purpose of feature?

Reply:
To be able to define async closure like ` my_function(async move | x, y | { })` [example here](https://github.com/dhall-mock/dhall-mock/pull/87/files#diff-7a09a902caff36d06b012fec16630f95dbe91e10403592507579db61fcf28fccR78)

Reply:
But I'm seeing that it break our CI so I'm going use another way